src/ensembl/src/content/app/browser/browser-region-editor/BrowserRegionEditor.tsx

```diff
@@ -79,6 +79,22 @@ export const BrowserRegionEditor = (props: BrowserRegionEditorProps) => {
   const [locationEndInput, setLocationEndInput] = useState(
     getCommaSeparatedNumber(locationEnd)
   );
+
+  const [shouldShowSubmitButton, showSubmitButton] = useState(false);
+
+  useEffect(() => {
+    if (
+      stickInput !== stick ||
+      getNumberWithoutCommas(locationStartInput) !== locationStart ||
+      getNumberWithoutCommas(locationEndInput) !== locationEnd
+    ) {
+      showSubmitButton(true);
+      return;
+    }
+
+    showSubmitButton(false);
+  }, [stickInput, locationStartInput, locationEndInput]);
```

Comment:
You may want to add the `stick` to the list of dependencies.

And change the code to:

```
const shouldShowButton = (
      stickInput !== stick ||
      getNumberWithoutCommas(locationStartInput) !== locationStart ||
      getNumberWithoutCommas(locationEndInput) !== locationEnd
    );
showSubmitButton(shouldShowButton);
```

Reply:
> You may want to add the stick to the list of dependencies.

Why do we need this?

Reply:
We don't strictly need this, but it's used within the useEffect callback, so if we were using linter rules for react hooks, the linter would complain.  🤷‍♂️

Reply:
Cool, that's done.